website/www/site/content/en/documentation/runtime/environments.md

```diff
@@ -17,147 +17,255 @@ limitations under the License.
-For example, `./gradlew :sdks:java:container:java8:docker -Pdocker-pull-licenses`. The files are added to `/opt/apache/beam/third_party_licenses/`.
-By default, no licenses/notices are added to the docker images.
-
-To examine the containers that you built, run `docker images` from anywhere in the command line. If you successfully built all of the container images, the command prints a table like the following:
-```
-REPOSITORY                         TAG                 IMAGE ID            CREATED           SIZE
-apache/beam_java8_sdk              latest              ...                 2 weeks ago       ...
-apache/beam_java11_sdk             latest              ...                 2 weeks ago       ...
-apache/beam_python2.7_sdk          latest              ...                 2 weeks ago       ...
-apache/beam_python3.5_sdk          latest              ...                 2 weeks ago       ...
-apache/beam_python3.6_sdk          latest              ...                 2 weeks ago       ...
-apache/beam_python3.7_sdk          latest              ...                 2 weeks ago       ...
-apache/beam_go_sdk                 latest              ...                 2 weeks ago       ...
-```
-
-### Overriding default Docker targets
-
-The default [tag](https://docs.docker.com/engine/reference/commandline/tag/) is sdk_version defined at [gradle.properties](https://github.com/apache/beam/blob/master/gradle.properties) and the default repositories are in the Docker Hub `apache` namespace.
-The `docker` command-line tool implicitly [pushes container images](#pushing-container-images) to this location.
-
-To tag a local image, set the `docker-tag` option when building the container. The following command tags a Python SDK image with a date.
-```
-./gradlew :sdks:python:container:py36:docker -Pdocker-tag=2019-10-04
-```
-
-To change the repository, set the `docker-repository-root` option to a new location. The following command sets the `docker-repository-root`
-to a repository named `example-repo` on Docker Hub.
-```
-./gradlew :sdks:python:container:py36:docker -Pdocker-repository-root=example-repo
-```
+python -m apache_beam.examples.wordcount \
+  --input gs://dataflow-samples/shakespeare/kinglear.txt \
+  --output "${GCS_PATH}/counts" \
+  --runner DataflowRunner \
+  --project $GCP_PROJECT \
+  --region $REGION \
+  --temp_location "${GCS_PATH}/tmp/" \
+  --experiment=use_runner_v2 \
+  --worker_harness_container_image=$IMAGE_URL
 
-## Pushing container images
+{{< /highlight >}}
 
-After [building a container image](#building-container-images), you can store it in a remote Docker repository.
 
-The following steps push a Python3.6 SDK image to the [`docker-root-repository` value](#overriding-default-docker-targets).
-Please log in to the destination repository as needed.
+### Troubleshooting/TIps
```

Comment:
Typo here--I also think you can just call this section "Troubleshooting" or "Considerations"